code/modules/mob/say.dm

```diff
@@ -37,7 +37,7 @@
 
 	message = strip_html_properly(message)
 
-	set_typing_indicator(0)
+	set_typing_indicator(0, TRUE)
```

Comment:
If state is a bool, please refactor it here as well

Reply:
It's kinda a boolean? I wasn't sure if I should use 'true' and 'false' there since it's meant to be 'on' or 'off'.

Reply:
You should